week-3/implement/is-proper-fraction.js

```diff
@@ -33,3 +33,68 @@
 // Explanation: The fraction 3/3 is not a proper fraction because the numerator is equal to the denominator. The function should return false.
 
 // These acceptance criteria cover a range of scenarios to ensure that the isProperFraction function handles both proper and improper fractions correctly and handles potential errors such as a zero denominator.
+
+function properFraction(num,den){
```

Comment:
a boolean is represented as `true` or `false` the letter case matter alot. they should all be in small letter

Reply:
i didn't know that, thank you